app/screens/apps_form/apps_form_component.tsx

```diff
@@ -269,21 +264,21 @@ function AppsFormComponent({
             }
         });
 
+        setError('');
```

Comment:
why is this needed?

Reply:
yep, I see, my mistake, there is a `setError` and a `setErrors`, I thought we were modifying the same state variable.